config/intellij-idea-inspections.xml

```diff
@@ -3027,6 +3027,16 @@
     <scope name="Tests" level="ERROR" enabled="false"/>
   </inspection_tool>
   <inspection_tool class="MagicNumberJS" enabled="true" level="ERROR" enabled_by_default="true"/>
+  <inspection_tool class="JUnitMalformedDeclaration" enabled="true" level="WARNING" enabled_by_default="true">
+    <option name="ignorableAnnotations">
+      <list>
+        <option value="mockit.Mocked" />
+        <option value="org.itsallcode.junit.sysextensions.SystemErrGuard.SysErr" />
+        <option value="org.itsallcode.junit.sysextensions.SystemOutGuard.SysOut" />
```

Comment:
For some reason, this suppression is not working. I will leave it here, in hopes that this is a bug, since IDEA will place violations on redundant suppressions and we can remove `@noinspection` in the future.